typescript/formsApps.d.ts

```diff
@@ -45,6 +45,28 @@ export type ApprovalsStyles = BaseFormsAppStyles
 export type FormsListStyles = BaseFormsAppStyles & {
   logoUrl?: string
   menuItems: FormsAppMenuItem[]
+  buttons?: {
+    submit?: {
+      icon?: string
+      label?: string
+    }
+    cancel?: {
+      icon?: string
+      label?: string
+    }
+    saveDraft?: {
+      icon?: string
+      label?: string
+    }
+    cancelPromptYes?: {
+      icon?: string
+      label?: string
+    }
+    cancelPromptNo?: {
+      icon?: string
+      label?: string
+    }
+  }
```

Comment:
Are we likely adding more buttons in future? If so, maybe we should organise these more like ```buttons.forms.cancelPrompt.yes```
```buttons.forms.submit```
?
Maybe just a matter of object structure preference.

Reply:
Only reason I would vote against having nested nested properties is it makes it harder to write a generic function to update/set them in the console. A function can be written for the property name quite easily if they all follow the same pattern e.g. 

```js
styles.buttons[propertyName].icon = ''
````

Reply:
Agreed, thats prob useful